api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -205,6 +201,26 @@ message ExternalProcessor {
 
   // Options related to the sending and receiving of dynamic metadata.
   MetadataOptions metadata_options = 16;
+
+  // If true, send each part of the HTTP request or response specified by ProcessingMode
+  // without pausing on filter chain iteration. It is "Send and Go" mode that can be used
+  // by external processor to observe Envoy data and status. In this mode:
+  // 1) Only STREAMED body processing mode is supported and any other body processing modes will be
```

Comment:
Does this render as an enumerated list in the docs output? In general, I think a bunch of the distinguished identifiers etc. in this comment block probably need some RST formatting.

Reply:
Sure. I have updated it to be rendered as an enumerated list.

Done